sale_payment_mgmt/models/account_payment.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
+
+    def _is_sale_payment_managed(self):
+        if self.env.user.has_group("sales_team.group_sale_salesman"):
+            return True
+        return False
+
+    def post(self):
+        obj = self
+        if self._is_sale_payment_managed() and not self.env.user.has_group(
```

Comment:
Include all conditions in the method, not half

Reply:
Sorry but i don't understand about it, because now, if user  `_is_sale_payment_managed` (user has group **sales_team.group_sale_salesman**) is necessary `sudo` to payment validation.

If user has group related to account NOT necessary apply `sudo`.

Reply:
What I mean is that you include the not part also on the method. This should only call the method.